nilearn/decoding/space_net.py

```diff
@@ -939,6 +940,9 @@ def predict(self, X):
             Predicted class label per sample.
         """
         # cast X into usual 2D array
+        if not hasattr(self, "masker_"):
+            raise NotFittedError("This %s instance is not fitted yet" % (
```

Comment:
This is the phrasing from sklearn.

On Sun, Nov 22, 2015 at 3:32 PM, Danilo Bzdok <notifications@github.com>
wrote:

> In nilearn/decoding/space_net.py
> <https://github.com/nilearn/nilearn/pull/851#discussion_r45559567>:
>
> > @@ -939,6 +940,9 @@ def predict(self, X):
> >              Predicted class label per sample.
> >          """
> >          # cast X into usual 2D array
> > +        if not hasattr(self, "masker_"):
> > +            raise NotFittedError("This %s instance is not fitted yet" % (
>
> rather "has not been fitted.."?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/851/files#r45559567>.
>



-- 
Elvis Dohmatob,
*sent from my  potato...*
http://dohmatob.blogspot.fr/, https://team.inria.fr/parietal/elvis/,
https://github.com/dohmatob